sdk/samples/core/mosquitto_rpc_server_sample.c

```diff
@@ -367,16 +366,17 @@ int main(int argc, char* argv[])
 
   az_mqtt5_rpc_server_codec_options server_codec_options
       = az_mqtt5_rpc_server_codec_options_default();
+  server_codec_options.service_group_id = service_group_id;
```

Comment:
Does this need to be specified if it's AZ_SPAN_EMPTY?

Reply:
Good point, removed it!